api/lib/domain/errors.js

```diff
@@ -6,6 +6,24 @@ class DomainError extends Error {
   }
 }
 
+class LocaleFormatError extends DomainError {
+  constructor(locale) {
+    super();
+    this.message = `Given locale is in invalid format: "${locale}".`;
```

Comment:
Détail : Supprimer le point à la fin de la string.

```suggestion
    this.message = `Given locale is in invalid format: "${locale}"`;
```